gym_ignition/runtimes/pybullet_runtime.py

```diff
@@ -214,6 +222,9 @@ def step(self, action: Action) -> State:
         if not self.action_space.contains(action):
             logger.warn("The action does not belong to the action space")
 
+        # Increase the counter
+        self.counter += 1
```

Comment:
Small suggestion: can we store the timestamp directly, and increase it by `1 / self.agent_rate` at every step? I don't think it will happen soon, but in this way the method will continue to work even if the agent_rate changes during the simulation.

Reply:
Sure, even better :+1: